### ISSUE TITLE
<p>The requested URL <code>/.well-known/caldav</code> was not found on this server

### ISSUE THREAD

**Issue by renellis** [pimutils/vdirsyncer#1059]
Hello,

vdirsyncer version 0.18.0

I'm trying to download google calendar event but I'm getting:

debug:   <p>The requested URL <code>/.well-known/caldav</code> was not found on this server.  <ins>That’s all we know.</ins>
error: Unknown error occurred: Not Found
error: Use `-vdebug` to see the full traceback.

Any idea what I should look at?

Thanks!

R.

**Comment by WhyNotHugo** [comment]
I assume that `my_calendars_local` is a filesystem collection not a caldav one, right?

**Comment by renellis** [comment]
> I assume that `my_calendars_local` is a filesystem collection not a caldav one, right?

Yes, I have omit other part of the config file as I don't think they are relevant. Btw, I running an older version, I will update and try again, it might be related to that.

**Comment by renellis** [comment]
After update to latest version I get the same error as in #1005

error: 400, message='Bad Request

**Comment by WhyNotHugo** [comment]
Try deleting the token_file and redoing the authentication flow.

**Comment by renellis** [comment]
The process seems to fail with the new version, after you approve the CalDAV app access, it redirect to a local addres: "http://127.0.0.1:33491/?state=9v......." and errored with This site can’t be reached127.0.0.1 refused to connect.

**Comment by WhyNotHugo** [comment]
Are you running vdirsycner on your local host?

Google deprecated the previous authentication flow we were using, so details for integrating with google have changed ([see these changes to the docs in particular](https://github.com/pimutils/vdirsyncer/commit/2138c434568a5b4e083a22efaa54a707f28be901)).

**Comment by WhyNotHugo** [comment]
Are you running vdirsycner on your local host?

Google deprecated the previous authentication flow we were using, so details for integrating with google have changed ([see these changes to the docs in particular](https://github.com/pimutils/vdirsyncer/commit/2138c434568a5b4e083a22efaa54a707f28be901)).
